test/bdd/features/introduce.feature

```diff
@@ -0,0 +1,1004 @@
+
```

Comment:
Tests must be succinct. These scenarios include a ton of details that must be removed. Eg. lines 16, 17, 18 (repeated throughout the file). And many others. These extra steps make it difficult to grasp the intent of each test.

Some quick resources:

* https://automationpanda.com/2017/01/30/bdd-101-writing-good-gherkin/
* https://www.testingexcellence.com/bdd-guidelines-best-practices/

Reply:
Check it now. Is that good or should I make it even smaller?

Reply:
Following is a list of my recommendations as per our chat. I've only reviewed the first scenario thus far, and I'm assuming these points apply to the rest:

* The steps in this feature file should be at a higher level of abstraction.
* We are not interested in "<agent> creates did exchange client". Nor are we interested in the details of how they exchange DIDs. In this file, all of that can be summed up to "Alice and Bob exchange DIDs"
* This is unnecessary: ""Alice,Bob,Carol" waits for public did to become available in sidetree for up to 10 seconds"
* We're also not interested in "<agent> creates introduce client"
* Since there is an implicit invitation in Carol's sidetree DID, I think it's worth pointing that out
* This is meaningless: ""Alice" continue with the introduce protocol".   I think it should be replaced with "Alice forwards Carol's implicit invitation to Bob" or something similar
* This should be removed: "Stop the introduce services and storage"